pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -716,8 +716,15 @@ def cmd_connection_save(context, name):
     connections = context.connections_repo
```

Comment:
For consistency with a similar change further down, I suggest to rename this variable "connections" also to "connections_repo".

May occur multiple times.

Reply:
DONE